base/multi.jl

```diff
@@ -2088,6 +2080,22 @@ function make_preduce_body(var, body)
     end
 end
 
+function pfor(f, R)
+    lenR = length(R)
+    chunks = splitrange(lenR, workers())
+    accums = get(task_local_storage(), :JULIA_ACCUMULATOR, ())
+    if accums !== ()
+        accums = accums[1]
```

Comment:
Wouldn't it be preferable to use a different variable name, since here (and bellow) `accums` is changing type?